addons/email_template/migrations/8.0.1.1/post-migration.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from openerp.openupgrade import openupgrade
+
+from openerp import SUPERUSER_ID
+from openerp.api import Environment
+
+
+_logger = logging.getLogger(__name__)
+
+
+@openupgrade.migrate()
+def migrate(cr, version):
+    if not version:
+        return
+    env = Environment(cr, SUPERUSER_ID, {})
+    convert_action_mail_server_email(env)
+
+
+def convert_action_mail_server_email(env):
+    def convert_body(body):
+        return body.replace("[[", "${").replace("]]", "}")
+
+    def convert_subject(subject):
+        subject = subject.strip()
+        if subject.startswith('object.'):
+            subject = "${%s}" % subject
+        return subject
+
+    actions = env['ir.actions.server'].search([
+        ('state', '=', 'email'),
+        ('template_id', '=', False),
+    ])
+
+    _logger.info(
+        'Transfering existing ir.actions.server emails to '
+        'email templates.'
+    )
+
+    for action in actions:
+
+        env.cr.execute(
```

Comment:
The values are preserved also, but columns are renamed to openupgrade_legacy.... It's a way to avoid that another module with the same field name waste old data.